multicluster/charts/linkerd-multicluster/values.yaml

```diff
@@ -26,6 +26,9 @@ gateway:
   # -- Set loadBalancerIP on gateway service
   loadBalancerIP: ""
 
+  # -- The pause container to use
+  pauseImage: "gcr.io/google_containers/pause:3.2"
```

Comment:
elsewhere in Linkerd we often use an `image:` struct with `name`, `version`, and `pullPolicy` fields.  is that worth doing here?

Reply:
I thought of it too, it's a really good point. I thought it's not necessarily worth doing from a maintenance perspective because it's unlikely we'd want people to run a different version or name the container differently; the pause container doesn't do much. For consistency, I think it'd be worth it. 

I took the route of least work here but if you think we should change it I'm all for it.

Reply:
I don't have a strong opinion, I'll leave it to your decision